src/components/errorMessages.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+interface Props {
+  message: string,
+  deleteErrorMessage: string,
+  isThereIssue: boolean,
+  editTodo: string,
+  setIsThereIssue: (value: boolean) => void,
+  isTitleEmpty: string,
+}
+
+export const Error: React.FC<Props> = ({
```

Comment:
Consider renaming the component from 'Error' to 'ErrorMessages' to better reflect its purpose.